dev/com.ibm.ws.jdbc/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -245,6 +245,8 @@ bulkLoadBatchSize=Bulk load batch size
 bulkLoadBatchSize.desc=JDBC driver property: bulkLoadBatchSize.
 bulkLoadOptions=Bulk load options
 bulkLoadOptions.desc=JDBC driver property: bulkLoadOptions.
+cancelSignalTimeout=Cancel Signal Timeout
```

Comment:
Noticed other strings are in sentence style (initial caps on first word only).  If it is the same type of UI element as connectNode or connectionWaitTimeout, then change to:
cancelSignalTimeout=Cancel signal timeout